LiveSplit.AutoSplitters.xml

```diff
@@ -20765,4 +20765,14 @@
         <Description>Autosplitter by tor1pp</Description>
         <Website>https://www.speedrun.com/simple_fish_adventure</Website>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Endling: Extinction is forever</Game>
```

Comment:
This list pulls an auto splitter only when the name in the splits editor matches exactly. That list is pulled from speedrun.com. The game's name there is `Endling Extinction Is Forever`.

```suggestion
            <Game>Endling Extinction Is Forever</Game>
```